### PR TITLE
Update remote git and helm example to use placement

### PR DESCRIPTION
**Pull request by lennysgarage**
* [x] I have taken backward compatibility into consideration.

Thought that since PlacementRule is being deprecated I'd start working on updating any examples that still use PlacementRule with placement.

### CODE REVIEW

File: examples/remote-git-sub/02-placement.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: nginx-demo-placement
+  namespace: nginx-app-ns
```
Comment (mikeshng) [inline examples/remote-git-sub/02-placement.yaml:5]: this namespace might not be correct

**Review by mikeshng — COMMENTED** [review]
double check the placement namespace please.

/hold

File: examples/remote-git-sub/02-placement.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: nginx-demo-placement
+  namespace: default
+spec:
+  numberOfClusters: 1
```
Comment (mikeshng) [inline examples/remote-git-sub/02-placement.yaml:7]:
```suggestion
  numberOfClusters: 10
```

lets just do 10, one seems too little

Comment (lennysgarage) [reply examples/remote-git-sub/02-placement.yaml]: 10 sounds good, updated it now

**Review by mikeshng — APPROVED** [review]
/lgtm

**Pull request merged**